tools/run_ganache_cli.sh

```diff
@@ -3,6 +3,6 @@
 script_dir_path="$(cd "$(dirname "$0")" && pwd)"
 
 "${script_dir_path}/../node_modules/.bin/ganache-cli" \
-    --accounts=300 \
+    --accounts=1000 \
```

Comment:
with ever more tests this will eventually not scale; i think we should reuse accounts in tests; i see no harm, but we can discuss

Reply:
Each `contract` section is reseting the accounts, basically, this 1000 accounts are only for one `contract` section. I have to increase it because of `enterCommittee` function test, where we have three positive cases, each having 300 validators. For example, in openst-contracts this value is set to 100.

As a summary: this approach does not block scaling, it eliminates possible subtle bags, it makes usage clean and easy.

As always, I am open to discuss this and any other approach.